tidy3d/components/heat/grid.py

```diff
@@ -9,8 +9,18 @@
 from ...exceptions import ValidationError
 
 
-class UniformUnstructuredGrid(Tidy3dBaseModel):
+class UnstructuredGrid(Tidy3dBaseModel):
+    """Abstract unstructured grid."""
+
+    relative_min_dl: pd.NonNegativeFloat = pd.Field(
+        1e-3,
+        title="Relative Mesh Size Limit",
+        description="The minimal allowed mesh size relative to the largest dimension of the simulation domain."
+        "Use 'relative_min_dl=0' to remove this constraint.",
```

Comment:
these ones use double backslash quotes for the docs RST 😭  it's a bit confusing just catching where I find this.

Reply:
yeah, even though I do remember which quotes to use where, these kind of things still slip in :'(